.prettierignore

```diff
@@ -0,0 +1 @@
+/yarn.lock
```

Comment:
this file will never be prettified anyway.. it's not supported.

Reply:
ok